### ISSUE TITLE
Increase "Add or Invite user via email" to h2

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#2657]
Hyku version: 6.1.0.0de435e845
Knapsack version: 0.0.1.5069cc809d

Dashboard - Tasks - Manage Users

The text "Add or invite user via email" should be a h2 and should be capitalized, e.g., "Add or Invite User via Email." 

In teaching librarians how to use the software, this section gets lost because the heading looks like text. Making it a h2 (since the top heading is a h1) would make it easier for library staff to find this section and immediately understand its use.

<img width="605" height="207" alt="Image" src="https://github.com/user-attachments/assets/55d0dcc8-6470-453c-8912-72efc39bd93d" />

**Comment by christhepianist** [comment]
In looking at other pages, I see that, instead of h2 tags, just bolding has been used to indicate a sub-heading (see Edit Groups area). To be consistent, this page should work the same way. Instead of h2, just bold the "Add or invite user via email."

Ultimately, however, these sub-headings should be HTML headings so that those using our site that have difficulty visually can hear the different sections of the page. This is done by using headings, not just formatting like bold.
